auth-api/src/auth_api/resources/org.py

```diff
@@ -85,12 +85,17 @@ def get():
         bcol_account_id = request.args.get('bcolAccountId', None)
         page = request.args.get('page', 1)
         limit = request.args.get('limit', 10)
+        validate_name = request.args.get('validateName', 'False')
 
         try:
             token = g.jwt_oidc_token_info
-            response, status = OrgService.search_orgs(business_identifier=business_identifier, access_type=access_type,
-                                                      name=name, status=status, bcol_account_id=bcol_account_id,
-                                                      page=page, limit=limit, token=token), http_status.HTTP_200_OK
+            if validate_name.upper() == 'TRUE':
+                response, status = OrgService.find_by_org_name(name), http_status.HTTP_200_OK
```

Comment:
If we return the full org , that could cause a problem.. anyone can access the orgs details using this API if they know the name...
Can we not return the full org details ? may be just return the status Or just the name

Reply:
It doesn’t rerun any information. Just blank one or none. The same as before.

Reply:
but its returning the OrgService.find_by_org_name(name) ? rite?

Reply:
I will just check the response of this call; if it doesnt have any org information ; I am good

Reply:
Response from service is processed at resource, based on role. Public user only see 200 and blank content {}, or 204 no content.

Reply:
thats good then